.github/workflows/ci.yml

```diff
@@ -87,7 +87,7 @@ jobs:
         if: steps.cache-dependencies.outputs.cache-hit != 'true'
 
       - name: Test
-        run: npx percy exec -- yarn test
+        run: npx percy exec -- npm run test
```

Comment:
Is there a reason this was changed from yarn?

Reply:
Yeah. `ember-website` uses `npm` instead of `yarn` to manage packages. I had initially written `yarn test` because I had copy-pasted the solution from `ember-octane-vs-classic-cheat-sheet`.